variables.tf

```diff
@@ -174,3 +174,9 @@ variable "terraformer_role_name" {
   description = "The name to assign the IAM role (and policy) that allows Terraformer instances to create appropriate AWS resources in this account."
   default     = "Terraformer"
 }
+
+variable "windows_with_docker" {
+  type        = bool
+  description = "A boolean to control the instance type used when creating Windows instances to allow Docker Desktop support."
```

Comment:
There should be a second sentence to the description stating that bare metal instance types are required to allow Docker Desktop support.  Without that key piece of information it's unclear to most folks what Docker Desktop has to do with the instance type.

Reply:
Please see 3d97ef5 and let me know if it is sufficient.